doc/tools/docgram/common.edit_mlg

```diff
@@ -2491,6 +2498,7 @@ RENAME: [
 | field_body field_spec
 | field_def field_val
 | bindings_with_parameters alias_definition
+| rewstrategy_prefix_op rewstrategy
```

Comment:
I am concerned about giving the appearance in the documentation that `rewstrategy` is for the atom and not the list.  Won't this confuse readers of Ltac2 tactic notations that use a rewstrategy scope?  CC @jfehrle

Reply:
As I suggested elsewhere, I'd use levels in rewstrategy (and drop this rename and the SPLICE above).